docs/cql/cql-extensions.md

```diff
@@ -133,21 +133,34 @@ token ranges.
 
 ## Synchronous materialized views
 
-Materialized view updates can be applied synchronously (with errors propagated
-back to the user) or asynchronously, in the background. Historically, in order
-to use synchronous updates, the materialized view had to be local,
-which could be achieved e.g. by using the same partition key definition
-as the one present in the base table.
-Scylla also allows explicitly marking the view as synchronous, which forces
-all its view updates to be updated synchronously. Such views tend to reduce
-observed availability of the base table, because a base table write would only
-succeed if all synchronous view updates also succeed. On the other hand,
-failed view updates would be detected immediately, and appropriate action
-can be taken (e.g. pruning the materialized view, as mentioned in the paragraph
-above).
-
-In order to mark a materialized view as synchronous, one can use the following
-syntax:
+Usually, when a table with materialized views is updated, the update to the
+views happen _asynchronously_, i.e., in the background. This means that
+the user cannot know when the view updates have all finished - or even be
+sure that they succeeded.
+
+In Scyla (but not in Cassandra), in some special cases the view updates
```

Comment:
s/Scyla/Scylla/ or ScyllaDB even

Reply:
We should use "ScyllaDB" when adding new content (there's another occurrence below).

Reply:
I'll fix the typo (of course) and add another patch which changes pre-existing instances of "Scylla" to "ScyllaDB".